data-prepper-core/src/main/java/com/amazon/dataprepper/peerforwarder/PeerForwardingProcessorDecorator.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazon.dataprepper.peerforwarder;
+
+import com.amazon.dataprepper.model.event.Event;
+import com.amazon.dataprepper.model.processor.Processor;
+import com.amazon.dataprepper.model.record.Record;
+import org.apache.commons.lang3.NotImplementedException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Collection;
+
+public class PeerForwardingProcessorDecorator implements Processor<Record<Event>, Record<Event>> {
+    private static final Logger LOG = LoggerFactory.getLogger(PeerForwardingProcessorDecorator.class);
+
+
+    @Override
+    public Collection<Record<Event>> execute(final Collection<Record<Event>> collection) {
+        LOG.info("Peer Forwarder not implemented yet, skipping events for now.");
+        throw new NotImplementedException();
```

Comment:
Java has a built-in exception you can use for now: `UnsupportedOperationException`. You can remove the Apache dependency by making this change.